app/views/catalog/_column_lists.html.haml

```diff
@@ -9,10 +9,12 @@
       %td{:align => "right", :valign => "top"}
         = select_tag("available_fields[]",
                      options_for_select(@edit[:new][:available_fields]),
-                     :multiple => true,
-                     :style    => "width: 450px",
-                     :size     => 8,
-                     :id       => "available_fields")
+                     :multiple                => true,
+                     'data-select-all-text'   => _('Select All'),
```

Comment:
@h-kataria I fixed the PR to add the select / deselect labels to selectpickers with `data-actions-box` set to true.